assets/sass/config/_variables.scss

```diff
@@ -65,6 +65,9 @@ $c-pagespeed-fast: $c-green-700;
 $c-pagespeed-average: $c-yellow-900;
 $c-pagespeed-slow: $c-red-600;
 
+// Idea Hub colors
+$c-idea-hub-orange: rgba(247, 143, 1, 0.2);
```

Comment:
We should probably keep color variables as normal hex codes without alpha values. We can always apply it with a different alpha, but the color itself should be just that.